modules/nw-ingress-sharding-dns.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the folllowing assemblies:
+//
+// * networking/ingress-sharding.adoc
+
+:_content-type: CONCEPT
+[id="nw-ingress-sharding-dns_{context}"]
+= Ingress sharding and DNS
+
+You need an external DNS server to route requests to the desired shard. Because of this, the cluster administrator is responsible for making a separate DNS entry for each router in a project. A router will not forward unknown routes to another router.
+
+Consider the following example:
+
+* Router A lives on host 192.168.0.5 and has routes with `*.foo.com`.
+* Router B lives on host 192.168.1.9 and has routes with `*.example.com`.
+
+Separate DNS entries must resolve `\*.foo.com` to the node hosting Router A and `*.example.com` to the node hosting Router B:
+
+* `*.foo.com A IN 192.168.0.5`
+* `*.example.com A IN 192.168.1.9`
```

Comment:
Where does this come from?  Using the asterisk in a DNS record is not correct.

Reply:
Another one from the 3.10 docs 🙂 https://docs.openshift.com/container-platform/3.10/install_config/router/default_haproxy_router.html#using-router-shards

Reply:
You can disregard my comment.